### PR TITLE
Chore: Added drawer extension e2e tests (Issue/3470)

### PR DESCRIPTION
**Pull request by lemmyadams**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
addresses adaptlearning/adapt-contrib-resources#111

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: test/e2e/extensions/drawer.cy.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import Course from '../../../src/course/en/course.json'
```
Comment (oliverfoster) [inline test/e2e/extensions/drawer.cy.js:1]: Interesting... very interesting...

The tests can be run against a specified build output folder, which wouldn't have a `src/course` but an `{outputdir}/{coursedir}/` instead. So we're going to need some way of grabbing that stuff in the test runtime such that the json can be loaded from the correct place.

Comment (oliverfoster) [reply test/e2e/extensions/drawer.cy.js:1]: A few people have mentioned they'd like to perform tests based upon the course content, effectively removing some Quality Assurance tasks. We should start from facilitating that point of view, even if we never get there. As the first test, this already relies on the defined course data.

Comment (oliverfoster) [reply test/e2e/extensions/drawer.cy.js:1]: The resource specific e2e tests should live in resources.

**Comment by joe-allen-89** [comment]
New data loading needs to be implemented.

**Comment by cahirodoherty-learningpool** [comment]
Replaced with https://github.com/adaptlearning/adapt-contrib-boxMenu/pull/168 and https://github.com/adaptlearning/adapt-contrib-resources/pull/112

**Pull request closed without merging**